inference-engine/ie_bridges/python/src/openvino/offline_transformations/offline_transformations_api_impl.cpp

```diff
@@ -26,16 +26,10 @@ void InferenceEnginePython::ApplyPOTTransformations(InferenceEnginePython::IENet
     manager.run_passes(network.actual->getFunction());
 }
 
-void InferenceEnginePython::ApplyLowLatencyTransformation(InferenceEnginePython::IENetwork network, int64_t num_iterations) {
+void InferenceEnginePython::ApplyLowLatencyTransformation(InferenceEnginePython::IENetwork network, bool use_const_initializer,
```

Comment:
I think we need to keep `ApplyLowLatencyTransformation` implementation as is (and deprecate this) and add new function `ApplyLowLatency2Transformation` for LowLatency2 transformation.

Reply:
Why do we need `ApplyLowLatencyTransformation `?